src/components/CommentCard/CommentCard.jsx

```diff
@@ -1,8 +1,35 @@
-const CommentCard = () => {
+import styles from "./CommentCard.module.css"
+import dateConverter from "../../utils/dateConverter";
+import { useEffect, useState } from "react";
+import { getUserInfo } from "../../utils/apis";
+
+const CommentCard = ({comment}) => {
+    const [commentAuthorInfo, setCommentAuthorInfo] = useState({});
+    const [isLoading, setIsLoading] = useState(true)
+    useEffect(() => {
+        setIsLoading(true)
+        getUserInfo(comment.author).then((userInfo) => {
+            setCommentAuthorInfo(userInfo);
+            setIsLoading(false);
+        })
+    }, [])
```

Comment:
Might be a good time to look into introducing a userContext so that you can store the user info there instead of needing a useEffect which calls the api each time this component is rendered.